### ISSUE TITLE
Paper link for 2d packing is broken

### ISSUE THREAD

**Issue by tataetae** [olragon/binpackingjs#29]
It is going to some site that has nothing to do with the algorithm. (I might be wrong, google translate was helping me out on the Finnish site). Is there any other source on the algorithm?

**Comment by olragon** [comment]
You're right, the original link is dead. The paper is "A Thousand Ways to Pack the Bin" by Jukka Jylänki. You can find it here:

- Archive.org mirror: https://web.archive.org/web/20210406100143/http://clb.demon.fi/files/RectangleBinPack.pdf  
- GitHub mirror: https://github.com/juj/RectangleBinPack (includes the PDF and reference C++ implementation)

We'll update the README link. Thanks for reporting!
